src/brucke_member.erl

```diff
@@ -215,7 +218,24 @@ handle_subscriber_down(#{ subscribers := Subscribers
     #subscriber{partition = Partition} = Subscriber ->
       lager:error("subscriber ~s:~p down, reason:\n~p",
                   [fmt_route(Route), Partition, Reason]),
-      NewSubscriber = Subscriber#subscriber{pid = ?DEAD(os:timestamp())},
+      NotPid =
+        case Reason of
+          normal ->
```

Comment:
is it always deleted topic when subscriber terminates with `normal` reason?

Reply:
yes it is.
in fact, it is perhaps a false 'UnknownTopicOrPartition',
second time seen this, there is perhaps a race condition in kafka during restart.
isn't much we can do though.